examples/bind/ecommerce.amp.html

```diff
@@ -5,59 +5,123 @@
+
+    /* custom css */
+    #header {
+      height: 24px;
+      padding: 16px;
+      margin: 0;
+      text-align: center;
+      text-transform: uppercase;
+      letter-spacing: 2px;
+      line-height: 24px;
+    }
+    .menu {
+      float: left;
+    }
+    .search {
+      float: right;
+    }
+    #container {
+      max-width: 500px;
+      padding: 32px 16px 64px 16px;
+      margin: auto;
+    }
+    .title {
+      text-align: center;
+    }
+    .rating {
+      margin-bottom: 0;
+      text-align: center;
+    }
+    .dots {
+      text-align: center;
+    }
+    .dots span {
+      display: inline-block;
+      background: #ccc;
+      border-radius: 4px;
+      width: 8px;
+      height: 8px;
+    }
+    .dots span.current {
+      background: #555;
+    }
+    .options {
+      margin: 16px 0;
     }
-    button {
+    .options h6 {
+      text-transform: uppercase;
+      margin: 0 0 4px 0;
+    }
+    .colors table {
+      margin: 0 -8px;
+    }
+    .colors amp-img {
+      border: solid 2px white;
+      display: block;
+      margin: auto;
+    }
+    .colors amp-img[selected] {
+      outline: solid 2px red;
+    }
+    .sizes div {
       border: solid 1px black;
+      width: 40px;
+      height: 40px;
+      line-height: 40px;
+      font-weight: 800;
+      text-align: center;
+    }
+    .sizes div[selected] {
+      background-color: gray;
+      color: white;
+      outline: none;
     }
-    amp-img {
-      border: solid 1px white;
+    .unavailable::before {
+      content: '';
+      position: absolute;
+      border-top: 1px solid black;
+      margin-left: 1px;
+      width: 57px;
+      transform: rotate(45deg);
+      transform-origin: 0% 0%;
+    }
+    .quantity select {
+      font-size: 16px;
+      border: solid 1px gray;
+      padding: 8px;
+    }
+    table {
+      width: 100%;
+    }
+    .hidden {
+      display: none;
     }
   </style>
 </head>
 
 <body>
-  <button onclick="AMP.toggleExperiment('amp-bind');window.location.href=window.location.href;">Toggle &lt;amp-bind&gt; experiment</button>
```

Comment:
Really there just for convenience, but `onclick` is not a valid AMP attribute. We can still use `AMP.toggleExperiment` in dev console.